treeherder/intermittents_commenter/commenter.py

```diff
@@ -385,6 +387,19 @@ def get_bug_run_info(self, bug):
             info.build_type = "unknown build"
         return info
 
+    def get_task_labels_and_count(self, manifest):
+        tasks_and_count = {}
+        summary_groups = (
+            fetch.get_summary_groups() if self.summary_groups is None else self.summary_groups
+        )
+        all_task_labels = summary_groups["job_type_names"]
+        for tasks_by_manifest in summary_groups["manifests"]:
+            for man, tasks in tasks_by_manifest.items():
+                if manifest == man:
+                    for task_index, status, _, count in tasks:
+                        tasks_and_count.setdefault(all_task_labels[task_index], {})[status] = count
```

Comment:
if we look at >1 day we could end up accumulating data here, so it will need to be a `+= count` and also assigned to `= 0` somewhere else.

in many ways we don't care about the `status`, mostly just number of jobs run :)  it is probably better to split on `status` because that can allow for other calculations, etc.